cli/src/feature.rs

```diff
@@ -14,6 +14,9 @@ use {
         input_validators::*, keypair::*,
     },
     solana_cli_output::{cli_version::CliVersion, QuietDisplay, VerboseDisplay},
+    solana_feature_gate_client::{
+        errors::SolanaFeatureGateError, instructions::RevokePendingActivation,
```

Comment:
nit: Outside the scope of this PR, but `FeatureGateError` would be a better name for the type